packages/lesswrong/server/emailComponents/EmailInactiveUserSurvey.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { registerComponent } from "../../lib/vulcan-lib";
+import { forumTitleSetting } from "../../lib/instanceSettings";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 16,
+    lineHeight: "22px",
+  },
+  link: {
+    color: theme.palette.primary.main,
+  },
+  hr: {
+    marginTop: 30,
+  }
+});
+
+const EmailInactiveUserSurvey = ({
+  user,
+  classes,
+}: {
+  user: DbUser;
+  classes: ClassesType;
+}) => {
+
+  return (
+    <div className={classes.root}>
```

Comment:
Did you test the preview of this email?

Reply:
(I know you're gonna edit it, and probably testing it after that is good too, but still, good to check the visuals. I would also include a way to send an actual test email to an admin account.)

Reply:
Yeah I tested it, and here is an updated version with the new copy:

![Screen Shot 2024-03-14 at 3 35 23 PM](https://github.com/ForumMagnum/ForumMagnum/assets/9057804/d361aeba-22c6-4809-894b-aa1e1ab481e5)

I'm not going to bother adding a way to send a test email to an admin account - the contents of this email are static, that functionality seems more useful to add for dynamic emails.